### ISSUE TITLE
NTR: Liver biopsy: inclusion bodies (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9060]
New term request Liver biopsy: inclusion bodies (f444): Hepatic biopsy

**Comment by pnrobinson** [comment]
@MickeySegal
Is this a finding that is specific to the liver or are these "general" inclusion bodies? Please provide more contaxt

**Comment by MickeySegal** [comment]
In https://www.ncbi.nlm.nih.gov/books/NBK1519/ they seem to be described only in liver:
"Liver pathology. AATD liver inclusions are visualized as bright pink globules of various sizes, using periodic acid-Schiff (PAS) stain following diastase treatment (PAS-D). The extent of inclusion formation varies considerably; the number and size of liver inclusions increases with age. Inclusions are not observed before age 12 weeks. Note: Liver biopsy, when indicated in the evaluation of individuals with liver disease, may show PAS positive diastase-resistant inclusion bodies which are suggestive of but not pathognomonic for AATD.

In infants with AATD, inclusions may be fine and granular and difficult to identify in percutaneous liver biopsy specimens. They are also observed in bile duct epithelium [[Cutz & Cox 1979](https://www.ncbi.nlm.nih.gov/books/NBK1519/#)].

Liver inclusions indicate the presence of at least one PI*Z allele; histologic examination of the liver cannot confidently distinguish between PI*MZ heterozygotes and PI*ZZ homozygotes, although inclusions are generally more profuse in PI*ZZ homozygotes. Visualization of inclusions may be variable among PI*MZ heterozygotes."

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000976
- label: Intrahepatic inclusion bodies
- parents: [Abnormal liver morphology (HP:0410042)](https://hpo.jax.org/app/browse/term/HP:0410042)
- pmid: 
- def: Nuclear or cytoplasmic aggregates of stainable substances within hepatocytes.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9060
- synonyms:
